src/python/WMCore/MicroService/Unified/MSOutput.py

```diff
@@ -397,8 +402,69 @@ def makeSubscriptions(self, workflow):
             pass
 
         elif self.msConfig['defaultDataManSys'] == 'Rucio':
-            pass
+            if isinstance(workflow, MSOutputTemplate):
+                msg = "Making transfer subscriptions for %s"
+                self.logger.info(msg, workflow['RequestName'])
 
+                rucioResultList = []
+                if workflow['destinationOutputMap']:
+                    for dMap in workflow['destinationOutputMap']:
+                        try:
+                            msg = "Making transfer subscriptions for %s"
+                            self.logger.info(msg, workflow['RequestName'])
+                            copies = workflow['numberOfCopies'] if workflow['numberOfCopies'] else 1
+                            # NOTE:
+                            #    Once we get rid of DDM this rseExpression generation
+                            #    should go in the Producer thread
+                            if workflow['isRelVal']:
+                                pnnExpr = '('+'|'.join(["pnn=%s" % pnn for pnn in dMap['destination']])+')'
```

Comment:
Is there any difference between `pnn` and `rse` in this expression? RucioInjector component injects data against an `rse`, so we might want to keep things consistent. Just for clarity, can you also expand that expression as a comment in plain english?

Reply:
I haven't notice any difference between them both so far. My only concern was that the `rse` may be more tricky to combine with additional requirements in the expression as ```  + '&cms_type=real&rse_type=DISK'``` And I may be totally wrong here btw. @nsmith- can you confirm ?

Reply:
Please use `rse=`.
`pnn` is only used for the syncing script in the case the RSE name != PNN name (essentially only true for `*_Tape` as pnn=`*_MSS`) and it might go away eventually

Reply:
Really, you don't even need `rse=`, for example:
```bash
$ rucio list-rses --expression "rse=T2_CH_CERN|rse=T2_US_MIT"
T2_CH_CERN
T2_US_MIT
$ rucio list-rses --expression "T2_CH_CERN|T2_US_MIT"
T2_CH_CERN
T2_US_MIT
```

Reply:
ok! Thanks @nsmith- Switching to `rse` then. I think I tried to add the additional constrain directly to SiteNames Unions and it was raising an exception for malformed `rseExpression`. I will give it a second chance though.

Reply:
It works [1]. I am going to use it like that now. Thanks @nsmith- 

[1]
```
In [45]: for i in rcl.list_rses('(T2_CH_CERN|T1_US_FNAL_Disk)&cms_type=real&rse_type=DISK'):
    ...:     pprint(i)

{u'rse': u'T2_CH_CERN'}
{u'rse': u'T1_US_FNAL_Disk'}
```